uint32 id
geometry_msgs/Pose pose
sensor_msgs/Image image